staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/validator.go

```diff
@@ -195,7 +196,8 @@ func (v *validator) Validate(ctx context.Context, versionedAttr *admission.Versi
 	}
 
 	options := cel.OptionalVariableBindings{VersionedParams: versionedParams}
-	auditAnnotationEvalResults, _, err := v.auditAnnotationFilter.ForInput(ctx, versionedAttr, cel.CreateAdmissionRequest(versionedAttr.Attributes), options, namespace, runtimeCELCostBudget)
+	//!TODO: Does this intentionally not use pre-existing var `admissionRequest`?
+	auditAnnotationEvalResults, _, err := v.auditAnnotationFilter.ForInput(ctx, versionedAttr, cel.CreateAdmissionRequest(versionedAttr.Attributes, metav1.GroupVersionResource(matchedResource), metav1.GroupVersionKind(versionedAttr.VersionedKind)), options, namespace, runtimeCELCostBudget)
```

Comment:
Does this intentionally not use pre-existing var `admissionRequest`?

Reply:
Nice catch! I don't thin it's intentionally since we don't mutate versionedAttr.

Reply:
change to use `admissionRequest`